R/tag_query.R

```diff
@@ -1023,7 +1023,8 @@ tagQuerySiblingRemove <- function(els) {
   tagQueryMatchChildRev(els, function(elParent, el, childPos) {
     # remove parent / child relationship
     el$parent <- NULL
-    elParent$children[[childPos]] <- NULL
+    # Remove the child entry completely
+    elParent$children[childPos] <- NULL
```

Comment:
Doesn't seems necessary

```suggestion
    elParent$children[[childPos]] <- NULL
```